src/quo2/components/wallet/keypair/view.cljs

```diff
@@ -25,69 +25,78 @@
        (if (= stored :on-device) (i18n/label :t/on-device) (i18n/label :t/on-keycard))))
 
 (defn avatar
-  [type full-name customization-color]
-  (if (= type :default-keypair)
-    [user-avatar/user-avatar
-     {:full-name           full-name
-      :ring?               true
-      :size                :small
-      :customization-color customization-color}]
-    [icon-avatar/icon-avatar
-     {:size    :size/s-32
-      :icon    :i/placeholder
-      :border? true}]))
+  [{:keys [type details customization-color]}]
+  (let [{:keys [full-name]} details]
+    (if (= type :default-keypair)
+      [user-avatar/user-avatar
+       {:full-name           full-name
+        :ring?               true
+        :size                :small
+        :customization-color customization-color}]
+      [icon-avatar/icon-avatar
+       {:size    :size/s-32
```

Comment:
just a note, we made a slight adjustment to the docs so that this will be `:size-32` after this pr is merged: 
https://github.com/status-im/status-mobile/pull/17279